docs/workWithSheet.md

```diff
@@ -0,0 +1,573 @@
+# Working with Sheet Operations
+
+[ [Overview](#overview) ]  [ [Operation details](#operation-details) ]  [ [Sample configuration](#sample-configuration) ]
+
+
+## Overview
+
+The following operations allow you to carry out sheet operations. Click an operation name to see details on how to use it.
+
+
+
+| Operation | Description |
+| ------------- |-------------|
+| [addSheetBatchRequest](#Add-Sheet-Using-BatchRequest) | Add a sheet or multiple sheets to a spreadsheet |
+| [deleteSheetBatchRequest](#Delete-Sheet-Using-BatchRequest) | Remove a sheet or multiple sheets from a given spreadsheet |
+| [getSheetMetaData](#Retrieve-Sheet-metadata) | Provides the sheet metadata within a given spreadsheet |
+| [updateSheetPropertiesBatchRequest](#Update-Sheet-Properties-Using-BatchRequest) | Update all sheet properties |
+| [copyTo](#Copy-Sheet-Data) | Copy a single sheet from a spreadsheet to another spreadsheet |
+
+
+## Operation details
+
+This section provides details on each of the operations.
+
+
+
+#### Add Sheet Using BatchRequest
+
+This method allow you to add new sheets to an existing spreadsheet. You can specify the sheet properties for the new sheet. It is an error to provide a title that is used for an existing sheet.
+
+
+
+###### addSheetBatchRequest
+
+```xml
+<googlespreadsheet.addSheetBatchRequest>
+     <spreadsheetId>{$ctx:spreadsheetId}</spreadsheetId>
+     <requests>{$ctx:requests}</requests>
+     <fields>{$ctx:fields}</fields>
+</googlespreadsheet.addSheetBatchRequest>
+```
+
+
+
+###### Properties
+
+* spreadsheetId :- Unique value of the spreadsheet
+* requests :- It contains data that is a kind of update to apply to a spreadsheet. To add multiple sheets within the spread sheet, need to repeat `addSheetBatchRequest` property within the `requests` attribute as below in the sample request.
+* fields [Optional] :- Specifying which fields to include in a partial response. For the following request only the `spreadsheetId` will be included in the response.
```

Comment:
Specify the fields that need to be included in a partial response. In the sample request given below, only the `spreadsheetId` will be included in the response.